crates/router/src/routes/subscription.rs

```diff
@@ -119,3 +137,85 @@ pub async fn confirm_subscription(
     ))
     .await
 }
+
+/// Add support for get subscription by id
+#[cfg(feature = "v1")]
```

Comment:
Is that mandatory to add feature flag here? because there is no version for subscription?

Reply:
better remove version here